news.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sorting-Algorithms-in-Javascript</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/77ebb379f6.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="/css/main.css">
```

Comment:
Ścieżki zaczynające się od `/` dobrze sprawdzają się, kiedy chcemy żeby strona działała tylko lokalnie na naszym komputerze, ale już gorzej jeśli spróbujemy ją wydeployować na jakiś serwer.  :) Serwery zwykle stoją na jakichś linuksach, a one ścieżki zaczynające się od `/` traktują jako absolutne i wskazujące na katalog główny - coś jak "C:" w Windowsie.

Zdecydowanie lepiej byłoby przerobić to na ścieżki względne z użyciem `./`, `../` itd.

Reply:
Ok, rozumiem. Poprawione. Tego samego schematu użyję w całym projekcie.